### ISSUE TITLE
Mouse_Lock_Target Issue

### ISSUE THREAD

**Issue by Bazz0r** [SunOner/yolov8_aimbot#152]
Hello, when setting mouse_lock_target to yes in the config file, it seems to disable the aimbot. Could you advise why this happens? Thank you.

**Comment by Bazz0r** [comment]
I see what is happening, this value does work however not when right click is held down. It seems to require a single right click and it seems to then automatically move to the target. However it does not lock to the target if the target moves, can this be achieved? For example, right click is help and when the mouse is moved the aim will not move from the detected target, even if the target moves?

**Comment by laaizy** [comment]
bro how you fix Runtime error?

**Comment by SunOner** [comment]
Disable mouse moving while target is locked?

**Comment by Bazz0r** [comment]
Yes if possible to a degree. In my testing, what I experience is that if the target is detected the mouse moves correctly to the target, however if the target then moves across the screen (for example a player running from right to left) the mouse movement does not track the player. I assumed perhaps incorrectly, the mouse_lock_target value would lock the mouse to the target until the target was no longer detected, i.e. either dead or obstructed from view. 

It would be great if something like this was possible, having the mouse either locked to the target and disabling the possibly of the user to move the mouse while the target is in view, or when the target is locked it requires an eratic mouse movement by the user to disengage the locking. I hope this makes sense and if any further detail or testing is required, please let me know.

**Comment by Geist4587** [comment]
Please please watch my videos I don't want any followers I need the help of the masses Bungie the Destiny 2 developers have been hacking my HP Omen PC for over a year I've asked for help everywhere Facebook Instagram YouTube and Twitter put me in incognito so the truth  It doesn't come out, the sad thing is I haven't even paid it off completely and they don't stop hacking me, please distribute my videos, help me, I've been fighting Bungie for a long time with depression.  https://youtube.com/shorts/feRDkmBOyA4?si=GoWKFnyxqIi-eWkg
